source/common/runtime/runtime_impl.cc

```diff
@@ -580,21 +593,21 @@ LoaderImpl::LoaderImpl(Event::Dispatcher& dispatcher, ThreadLocal::SlotAllocator
                                            Filesystem::Watcher::Events::MovedTo,
                                            [this](uint32_t) { return loadNewSnapshot(); });
       if (!creation_status.ok()) {
```

Comment:
this can turn to RETURN_IF_NOT_OK()

actually RETURN_IF_NOT_OK around addWatch and kill off creation_status temporary

Reply:
RETURN_IF_NOT_OK can't do that. I'm fixing it in a separate PR